jkube-kit/doc/src/main/asciidoc/inc/resource-generation/_controller_resource_generation.adoc

```diff
@@ -11,6 +11,7 @@ In JKube terminology, a Controller resource is a Kubernetes resource which manag
 * https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/[StatefulSet]
 * https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/[DaemonSet]
 * https://kubernetes.io/docs/concepts/workloads/controllers/job/[Job]
+* https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/[CronJob]
```

Comment:
I think we need to add new field `schedule` to `.controller fields for configuring generated controllers` table